src/barril/units/posc.py

```diff
@@ -321,6 +321,7 @@ def FillUnitDatabaseWithPosc(db=None, fill_categories=True, override_categories=
         "moment per angular velocity", "newton meter per angular velocity", "Nms/rad"
     )
     db.AddUnitBase("mass temperature per mol", "kg.K/mol", "kg.K/mol")
+    db.AddUnitBase("temperature per pressure", "K/Pa", "K/Pa")
```

Comment:
Consider calling it:

```
db.AddUnitBase("joule-thomson coefficient", "delta kelvin per pascal", "K/Pa")
```

or, if you want to call it literally, perhaps it's better to use "delta temperature per pressure"

(remember to also change in the `db.AddUnit(...)` calls below)

Reply:
Nice, thanks!